internal/migrations/20210822134659_multiple_networks.go

```diff
@@ -0,0 +1,67 @@
+package migrations
+
+import (
+	"github.com/jinzhu/gorm"
+	"github.com/openshift/assisted-service/internal/common"
+	"github.com/openshift/assisted-service/models"
+	gormigrate "gopkg.in/gormigrate.v1"
+)
+
+func multipleNetworks() *gormigrate.Migration {
+	migrate := func(tx *gorm.DB) error {
+		dbClusters, err := common.GetClustersFromDBWhere(tx, common.UseEagerLoading, common.IncludeDeletedRecords)
+		if err != nil {
+			return err
+		}
+		for _, cluster := range dbClusters {
+			if cluster.ClusterNetworkCidr != "" {
```

Comment:
what will happen to `cluster.ClusterNetworkCidr` and all the other old fields, shouldn't they be deleted?

Reply:
Actually, they can get deleted. But I thought it would be easier to make this transition gradual to keep the old fields and later on to delete them

Reply:
The api update both?

Reply:
yes.
https://github.com/openshift/assisted-service/pull/2455